frontend/lib/generators/spree/frontend/install/install_generator.rb

```diff
@@ -16,9 +16,6 @@ def self.source_paths
         end
 
         def install
-          # main navigation configuration
-          template 'config/initializers/spree_storefront.rb', 'config/initializers/spree_storefront.rb'
-          template 'config/spree_storefront.yml', 'config/spree_storefront.yml'
```

Comment:
This is fine :) For new installations we want to dispose of those files, we can also remove all of the https://github.com/spree/spree/tree/master/frontend/app/assets/images/meganav